doc/faq.rst

```diff
@@ -42,6 +42,25 @@ Installing and updating
       :alt: Navigator showing installing specific version of Spyder
 
 
+.. dropdown:: Q: Is commercial use allowed? What's the situation with Anaconda?
+
+   Spyder is 100% free and open source; there is no paid version or prohibition on commercial use.
+   Spyder is developed by its international user community, and supported by its users through `OpenCollective`_ and by its generous sponsoring organizations, including `Quansight`_ and `NumFOCUS`_.
+   Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) are fully unencumbered and can be freely redistributed, used and modified by anyone.
```

Comment:
I would suggest replacing the "unencumbered" word for something more familiar (specially for non native speakers).